### DISCUSSION TITLE
Player crashing but nothing obvious on log?

### DISCUSSION THREAD

**Discussion by buzzcut121** [smplayer-dev/smplayer#268] [General]
I'm running the player on a new windows10 laptop.

After I installed the player, it worked fine. After a while, when I might select a new file to play, the player opens but freezes every time. 

The tail-end of the log follows but I can't see any obvious problem.

METADATA_DATE=
METADATA_TRACK=
METADATA_COPYRIGHT=
INFO_MEDIA_TITLE=Wakefield.S01E01.1080p.WEB-DL.AAC2.0.H.264-WH.mkv
INFO_STREAM_PATH=C:/Users/brend/Downloads/TV TO WATCH/Wakefield - Season 1/Wakefield.S01E01.1080p.WEB-DL.AAC2.0.H.264-WH.mkv
INFO_TRACK_0: video 1 '' '' yes
INFO_TRACK_1: audio 1 'eng' '' yes
INFO_TRACK_2: sub 1 'eng' '' yes
[ao/wasapi] Under-run: Device delay: -0.4 us

Any error would probably show up on the log just before the end, right?

Things I have tried to fix the problem include rebooting & reinstalling SMplayer. Also, I had a skin applied (Black,Default) and when I switched back to the basic player, the files worked again. Any ideas as to what's going wrong here?

**Comment by smplayer-dev** [comment]
Yes, the log doesn't provide useful information. 
Try selecting different video and audio outputs in preferences -> general, just in case it's a drivers problem.
